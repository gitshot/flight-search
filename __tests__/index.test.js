import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../pages/index.js';

describe('index', () => {
  it('Index shows search results', () => {
    const shows = [
      {
        show: {
          id: 'show-1',
          name: 'show name',
        },
      },
    ];
    const component = renderer.create(<Index shows={shows} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
