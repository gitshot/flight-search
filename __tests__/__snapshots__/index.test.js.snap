// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index Index shows gists 1`] = `
<div
  className="Layout-s1i2z5um-1 flAUel"
>
  <div
    className="Header-duumzn-2 iWjsxc"
  >
    <h1
      className="Header__Title-duumzn-1 ZeEZy"
    >
      <a
        href="/"
        onClick={[Function]}
      >
        <strong>
          gist
        </strong>
        <br />
        viewer
      </a>
    </h1>
    <nav>
      <a
        className="Header__Anchor-duumzn-0 dIukbg"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="Header__Anchor-duumzn-0 dIukbg"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        className="Header__Anchor-duumzn-0 dIukbg"
        href="/flights"
        onClick={[Function]}
      >
        Flights
      </a>
    </nav>
  </div>
  <div
    className="Layout__Content-s1i2z5um-0 kyjkTw"
  >
    <div
      className="Profile-s1vsabj8-2 dnYTKS"
    >
      <h1
        className="Profile__Heading-s1vsabj8-1 bDIBql"
      >
        next guy
      </h1>
      <img
        className="Profile__Photo-s1vsabj8-0 fZpJFQ"
        src="url"
      />
    </div>
    <ul>
      <li>
        <a
          href="/gist/gist-1"
          onClick={[Function]}
        >
          filename
        </a>
      </li>
    </ul>
  </div>
</div>
`;
